ledger/metrics/src/main/scala/com/daml/metrics/IndexerMetrics.scala

```diff
@@ -53,7 +53,8 @@ class IndexerMetrics(prefix: MetricName, factory: MetricsFactory) {
                     |can be negative.""",
     qualification = Debug,
   )
-  val currentRecordTimeLag: Gauge[Long] = DropwizardGauge(prefix :+ "current_record_time_lag", null)
+  val currentRecordTimeLagForDocs: Gauge[Long] =
```

Comment:
Just wondering, what prevents us from defining this at the place where it actually gets updated?

Reply:
We didn't have support for the async gauge as of yet as it wasn't  returning anything. If we were to make `CloseableGauge` a metric handle we wouldn't require this anymore.

Reply:
Will handle that in a separate PR